<!-- 
  Launch:  
-->
<launch>
	<include file="$(find osm_server)/launch/osm_path_generator_servis.launch"/>
	<!--node pkg="osm_servis" type="gps_kalman_filter.py" name="gps_reader" output="screen"/-->
	<node pkg="osm_server" type="osm_path_gps.py" name="osm_path" output="screen"/>

          <param name="lat1" value="52.2408" />
          <param name="lat2" value="52.2052" />
          <param name="lng1" value="16.2620" />
          <param name="lng2" value="16.2264" />

          <param name="aeroway_filter" value="1.0" />

          
</launch>
